### PR TITLE
District wise daily cases and testing count Python scripts  and corresponding json files

### PR DESCRIPTION
**Pull request by Abhishekbhatia**
Python scripts for daily district wise number of cases and testing count and the corresponding json files

### CODE REVIEW

**Comment by JunaidBabu** [comment]
We're planning to add num_cases to v1 and v2 of raw_data as well. Will review this PR once again after the format changes.

File: src/python_scripts/district_level_daily_cases.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import requests
+import json
+import sys,os
```
Comment (Manik-Jain) [inline src/python_scripts/district_level_daily_cases.py:3]: never used

File: src/python_scripts/district_level_daily_cases.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import pdb
```
Comment (Manik-Jain) [inline src/python_scripts/district_level_daily_cases.py:4]: never used

**Comment by JunaidBabu** [comment]
This has already been implemented and the data is available in v3 and v4. If there are more requirements kindly raise an issue and let's discuss first.

Closing this.

**Pull request closed without merging**